docs/en/how-to-get-started.md

```diff
@@ -48,13 +48,13 @@ Once the client gets instantiated, we must force personal sign by pretending to
 Next, let's setup a a signer and then get the `callerAddress` (that is your address on Ethereum or Rinkeby):
 
 ```js
-  async _setupSigner (plasmaClient, provider) {
+  async _setupSigner (basechainClient, provider) {
```

Comment:
All these variables etc. should be named `loomWhatever` since this code is meant to work with both Extdev and Basechain, right?

Reply:
You're right. Thanks!